app/forms/steps/income/income_payment_fieldset_form.rb

```diff
@@ -11,7 +11,8 @@ class IncomePaymentFieldsetForm < Steps::BaseFormObject
       validate { presence_with_payment_type :frequency }
 
       validates :amount, numericality: {
-        greater_than: 0
+        greater_than: 0,
+        less_than_or_equal_to: 9_999_999_999.99
```

Comment:
Obviously a couple of these get combined to inject into the 'Other income' field along with trust fund dividend - I didn't know a good way to decide what to reduce it to, so left as max